scalyr_agent/log_line_debug.py

```diff
@@ -20,6 +26,24 @@
 # the kubernetes monitor will write the id of the needed contained in this variable.
 CONTAINER_ID = None
 
+file_logger = None
+
+original_set_log_destination = scalyr_logging.set_log_destination
+
+def scalyr_logging_set_log_destination(*args, **kwargs):
+    global file_logger
+    original_set_log_destination(*args, **kwargs)
+    file_logger = logging.getLogger("PAYLOAD")
+
+    path = os.path.join(
+        tempfile.gettempdir(),
+        "debug_payload.log"
+    )
+    file_handler = logging.FileHandler(filename=path)
```

Comment:
I don't think, by default, that we get a rotating log file handler.  Can you change to use rotation please?  Let's keep 3 rotations and cap each file at 50MB.